flink-filesystems/flink-s3-fs-presto/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,27 @@
+This project includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+  - com.facebook.presto presto-hive version 0.185
+  - com.facebook.presto.hadoop hadoop-apache2 version 2.7.3-1
```

Comment:
are we ignoring the contained dependencies, or are assuming that they bundle all licenses correctly?

Reply:
I would assume that they are doing this correctly. Otherwise we might be screwed anyway, because we would have to check this for every dependency which is not really doable.